### PR TITLE
patches: new TEKGREN textures by @KorpKat

### PR DESCRIPTION
**Pull request by Talon1024**

### CODE REVIEW

**Comment by mc776** [comment]
My comments on #1149 actually had these in mind as well.

File: lumps/textures/textures.cfg
Comment (selliott512) [inline lumps/textures/textures.cfg:1417]: I don't understand these comments since they are all 64 x 128, but that's ok.

**Review by selliott512 — DISMISSED** [review]
Looks good.

I have a soft spot for bright clean textures in video games, which this seems to be. I like the little light, for example. But I know that may be a topic of some controversy. Some may prefer dark and gloomy textures.

File: lumps/textures/textures.cfg
Comment (Talon1024) [reply lumps/textures/textures.cfg:1417]: Those comments describe the shape of the panels on those patches. Also, I had to use something other than a number for the suffix because `TEKGREN10` doesn't fit into 8 characters.

**Comment by Talon1024** [comment]
Before this gets merged, I recently tried this out ingame, and it seems to be... Blander than the previous TEKGREN set.
Screenshot_Doom_20231029_020836
Screenshot_Doom_20231029_021259

**Comment by mc776** [comment]
> Before this gets merged, I recently tried this out ingame, and it seems to be... Blander than the previous TEKGREN set. Screenshot_Doom_20231029_020836 Screenshot_Doom_20231029_021259

I agree, and despite Steven's approval I do not intend to be the person who pushes the green button on this.

**Comment by mc776** [comment]
I'm also against the idea of turning this set of textures into a mere "METAL2 light mode" rather than leaving it to be its own thing - it just makes the texture set as a whole less varied, to no apparent advantage.

**Comment by mc776** [comment]
And on a purely aesthetic level the change to RW37_4 is a pure downgrade - poorer balance in shapes and positive/negative space, increased contrast clashes and makes the pipes feel more pasted-on than the old ones.

**Comment by mc776** [comment]
pipe1
pipe2
blue1
blue2
4a1
4a2
4b1
4b2

It just feels like going from Quake to Chex Quest for me...

**Comment by mc776** [comment]
As with the METAL2 replacements, the resulting textures are *much* less visually forgiving of any deviation from the standard texturer-intended alignment, making anything look worse that might have played with any of the looser tolerances of the original, like with that curve in Map04.

This implicitly pushes people towards sticking with square angles and wall lengths of multiples of 32 or 64, removing much of the discretion for unusual shapes that was one of the really big things that Doom improved over its predecessors.

**This change is detrimental to mappers.**

As I've already got my own fork running I can of course just defer to majority opinion if everyone else thinks this change is awesome and great, but again I think this makes Freedoom worse.

**Comment by Big-the-Fat** [comment]
Agreed.  In isolation the newer textures look fine but in-game they are stark and as shown above less flexible.

**Pull request closed without merging**
